<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClassificacaoWindow</class>
 <widget class="QMainWindow" name="ClassificacaoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>381</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Wavelets</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Detalhes Experimento</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Seletor</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0" colspan="3">
          <widget class="QLineEdit" name="leDescricao"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Dataset</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="4">
          <widget class="QComboBox" name="cbDataset"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Detalhe</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Classificador</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QComboBox" name="cbDetalhe">
           <item>
            <property name="text">
             <string>Todos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Aproximacao</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Horizontais</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Verticais</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Diagonais</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Kernel</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Funcao</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="3">
          <widget class="QComboBox" name="cbClassificador">
           <item>
            <property name="text">
             <string>svm</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>tree</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1" colspan="3">
          <widget class="QComboBox" name="cbKernel">
           <item>
            <property name="text">
             <string>todos</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>linear</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>rbf</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>poly</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="cbFuncao">
           <item>
            <property name="text">
             <string>calcmatrix</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Folds ou % p/ teste</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Select</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QComboBox" name="cbSeletor">
           <item>
            <property name="text">
             <string>anova</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>pca</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="7" column="2" colspan="2">
          <widget class="QLineEdit" name="leSelect"/>
         </item>
         <item row="7" column="1">
          <widget class="QLineEdit" name="leFolds"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Descrição</string>
           </property>
          </widget>
         </item>
         <item row="11" column="3">
          <widget class="QToolButton" name="tbDesc">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Output</string>
           </property>
          </widget>
         </item>
         <item row="13" column="0" colspan="3">
          <widget class="QLineEdit" name="leFileResult"/>
         </item>
         <item row="13" column="3">
          <widget class="QToolButton" name="tbOutput">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>252</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btIncluir">
         <property name="text">
          <string>Salvar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btExecutar">
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="text">
          <string>Encerrar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>380</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
